### ISSUE TITLE
Graphviz SVGs cause exception

### ISSUE THREAD

**Issue by ismay** [ismay/coarse#13]
(@szacchino Moved your issue to the library that's causing the error)

---

Hi,

trying with several SVG as generated by dot/graphviz (i.e.: https://graphviz.gitlab.io/_pages/Gallery/directed/cluster.svg) causes the following exception

```
/usr/local/lib/node_modules/coarse-cli/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:206
          throw new DOMException("The object can not be found here.", "NotFoundError");
          ^
NotFoundError: The object can not be found here.
    at SVGSVGElementImpl.insertBefore (/usr/local/lib/node_modules/coarse-cli/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:206:17)
    at SVGSVGElementImpl.replaceChild (/usr/local/lib/node_modules/coarse-cli/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:269:10)
    at SVGSVGElement.replaceChild (/usr/local/lib/node_modules/coarse-cli/node_modules/jsdom/lib/jsdom/living/generated/Node.js:307:45)
    at coarse (/usr/local/lib/node_modules/coarse-cli/node_modules/coarse/lib/index.js:102:11)
    at fn (/usr/local/lib/node_modules/coarse-cli/bin/index.js:71:32)
    at Object.<anonymous> (/usr/local/lib/node_modules/coarse-cli/bin/index.js:75:1)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
```

I've just used `coarse <input> <output>` without any further options.

---

Moved from https://github.com/ismay/coarse-cli/issues/2

**Comment by szacchino** [comment]
As far as I can debug, it seems that the call to `replaceChild()` causes the error. If I replace `svg.replaceChild(replacement, original);` with `svg.appendChild(replacement);` it exits with success. However, opening the output with gravit you can see that the coarse elements are all outside the viewbox or have a nan/0 border or a transparent color but they are there. So it seems that the error is due to DOM manipulation and not on drawing conversion per-se.

**Comment by ismay** [comment]
Ok, I'll see if I can find out what's going wrong exactly.

**Comment by ismay** [comment]
Released as 0.8.1
